Source/Core/Core/Analytics.cpp

```diff
@@ -116,6 +118,61 @@ void DolphinAnalytics::ReportGameStart()
   Send(builder);
 }
 
+void DolphinAnalytics::ReportUnknownIOSDevice(const std::string& path)
+{
+  Common::AnalyticsReportBuilder builder = MakeBuilderWithGameID();
+  builder.AddData("type", "ios-unknown-device");
+  builder.AddData("ios-version", IOS::HLE::GetVersion());
+  builder.AddData("ios-path", path);
+  Send(builder);
+}
+
+void DolphinAnalytics::ReportUnknownIOSRequest(const std::string& path,
```

Comment:
I put it there to be somewhat consistent with ReportGameStart. However, it's true that this adds a dependency from analytics to IOS... Would you prefer if this is moved to near the IOS code instead of being here? (I don't really mind either way.)